prepare/metrics/grouped_instance_metrics.py

```diff
@@ -0,0 +1,553 @@
+)
+
+add_to_catalog(
+    metric,
+    "metrics.robustness.fixed_group_norm_hedges_g_paraphrase_accuracy",
+    overwrite=True,
+)
+
+
+metric = FixedGroupNormHedgesGParaphraseStringContainment()
+global_target = {
+    "fixed_group_norm_hedges_g_paraphrase_string_containment": -0.08,
+    "score": -0.08,
+    "score_name": "fixed_group_norm_hedges_g_paraphrase_string_containment",
+    "score_ci_low": -0.1,
+    "score_ci_high": -0.05,
+    "fixed_group_norm_hedges_g_paraphrase_string_containment_ci_low": -0.1,
+    "fixed_group_norm_hedges_g_paraphrase_string_containment_ci_high": -0.05,
+}
+
+
+outputs = test_metric(
+    metric=metric,
+    predictions=predictions,
+    references=references,
+    instance_targets=instance_targets_string_containment,
+    global_target=global_target,
+    additional_inputs=additional_inputs,
+)
+
+add_to_catalog(
+    metric,
+    "metrics.robustness.fixed_group_norm_hedges_g_paraphrase_string_containment",
+    overwrite=True,
+)
+
+# TokenOverlap: example of a metric that has more than one score
+
+global_target = {
+    "group_mean_f1": 0.51,
+    "score": 0.51,
+    "score_name": "group_mean_f1",
+    "group_mean_f1_ci_low": 0.22,
+    "group_mean_f1_ci_high": 0.68,
+    "score_ci_low": 0.22,
+    "score_ci_high": 0.68,
+    "group_mean_precision": 0.5,
+    "group_mean_precision_ci_low": 0.21,
+    "group_mean_precision_ci_high": 0.67,
+    "group_mean_recall": 0.52,
+    "group_mean_recall_ci_low": 0.25,
+    "group_mean_recall_ci_high": 0.71,
+}
+
+instance_targets_token_overlap = [
+    {"precision": 1.0, "recall": 1.0, "f1": 1.0, "score": 1.0, "score_name": "f1"},
+    {"precision": 1.0, "recall": 1.0, "f1": 1.0, "score": 1.0, "score_name": "f1"},
+    {"precision": 1.0, "recall": 1.0, "f1": 1.0, "score": 1.0, "score_name": "f1"},
+    {"precision": 0, "recall": 0, "f1": 0, "score": 0, "score_name": "f1"},
+    {"precision": 0, "recall": 0, "f1": 0, "score": 0, "score_name": "f1"},
+    {"precision": 1.0, "recall": 1.0, "f1": 1.0, "score": 1.0, "score_name": "f1"},
+    {"precision": 1.0, "recall": 1.0, "f1": 1.0, "score": 1.0, "score_name": "f1"},
+    {"precision": 1.0, "recall": 1.0, "f1": 1.0, "score": 1.0, "score_name": "f1"},
+    {"precision": 1.0, "recall": 1.0, "f1": 1.0, "score": 1.0, "score_name": "f1"},
+    {"precision": 0.5, "recall": 1.0, "f1": 0.67, "score": 0.67, "score_name": "f1"},
+    {"precision": 1.0, "recall": 1.0, "f1": 1.0, "score": 1.0, "score_name": "f1"},
+    {"precision": 1.0, "recall": 1.0, "f1": 1.0, "score": 1.0, "score_name": "f1"},
+    {"precision": 0, "recall": 0, "f1": 0, "score": 0, "score_name": "f1"},
+    {"precision": 0, "recall": 0, "f1": 0, "score": 0, "score_name": "f1"},
+    {"precision": 0, "recall": 0, "f1": 0, "score": 0, "score_name": "f1"},
+]
+
+
+metric = GroupMeanTokenOverlap()
+
+outputs = test_metric(
+    metric=metric,
+    predictions=[str(vv) for vv in predictions],
+    references=[[str(vvv) for vvv in vv] for vv in references],
```

Comment:
why the list comprehension loop?

Reply:
To make them strings which TokenOverlap expects, but I'm going to do the string conversion within TokenOverlap instead